lv_conf_template.h

```diff
@@ -1131,6 +1137,16 @@
 /** LVGL Windows backend */
 #define LV_USE_WINDOWS    0
 
+/** LVGL UEFI backend */
+#define LV_USE_UEFI 0
+#if LV_USE_UEFI
+    #define LV_USE_UEFI_INCLUDE "myefi.h"   /**< Header that hides the actual framework (EDK2, gnu-efi, ...) */
```

Comment:
Can we create some headers for the most typical systems?

Reply:
That would be edk2 and gnu-efi, both require just a few includes. I think that most users will use a single include for LV_USE_UEFI_INCLUDE and LV_XXX_INCLUDE, so I would prefer to list the needed includes in a comment inside lv_uefi.h.

Reply:
If the list of headers are fixed can we add ready-to-use headers? E.g. `lv_uefi_edk2.h` and `lv_uefi_gnu_efi.h`?

Reply:
I added 3 new headers that should be enough for most users:
- src/drivers/uefi/lv_uefi_edk2.h: a wrapper for edk2
- src/drivers/uefi/lv_uefi_gnu_efi.h: a wrapper for gnu-efi
- src/drivers/uefi/lv_uefi_std_wrapper.h: a wrapper on top of the other two libraries that can be used to define the LV_XXX_INCLUDES